orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/AmazonImageFinder.java

```diff
@@ -83,8 +83,10 @@ public boolean test(AmazonImage amazonImage) {
       Map<String, String> tags,
       List<String> warningsCollector) {
     StageData stageData = (StageData) stage.mapTo(StageData.class);
+
     List<AmazonImage> allMatchedImages =
-        oortService.findImage(getCloudProvider(), packageName, null, null, prefixTags(tags))
+        oortService
+            .findImage(getCloudProvider(), packageName, stageData.account, null, prefixTags(tags))
```

Comment:
LGTM, @cfieber do you see any reason to avoid passing the account to clouddriver if we have it?